utils/display/apps.go

```diff
@@ -37,10 +33,13 @@ func AppsTable(apps []*meroxa.Application, hideHeaders bool) string {
 
 		env, err := app.Environment.GetNameOrUUID()
 
-		if err == nil && itShouldEnvInfo(app.Environment) {
+		if err == nil {
 			withEnvironment = true
 			r = append(r, &simpletable.Cell{Align: simpletable.AlignLeft, Text: env})
 		}
+		if withEnvironment == true && err != nil {
+			r = append(r, &simpletable.Cell{Align: simpletable.AlignLeft, Text: "common"})
+		}
```

Comment:
Logic here seems a little weird
I feel like we can also check if env is nil with `app.Environment != nil` for whether the app is on a common env vs not

Reply:
So, in this I'm trying to cover a case  when we do add an Environment column on table for an app with a private env, but there is also an app without environment (aka common). 
If app is common, app.Environment will be nil , but we would still want to add a text as the simple table starts panicking when Environment column exists but we pass it no value in a row 🤔

Reply:
Yeah, but the only time `withEnvironment` is set to true if `err == nil`, so the second if statement will never work

Reply:
Looking at it again, ye it makes no sense. Made some changes. We are now adding Env column no matter what and just filling it with common if an app doesn't have a private app